starter-repos/app-template/skaffold-ci.yaml

```diff
@@ -11,8 +10,19 @@ profiles:
   - name: ci
     build:
       cluster:
+        # Note: `skaffold build` may fail with
+        # 'secrets "docker-cfg" already exists' error.
+        # If that happens, do not use "dockerConfig 1"
+        # but "dockerConfig 2" as a workaround.
+
+        # dockerConfig 1
         dockerConfig:
-          path: ~/.docker/config.json          
+          path: ~/.docker/config.json
+
+        # # dockerConfig 2
+        # dockerConfig:
+        #   secretName: docker-cfg
```

Comment:
I've tried this, but this might return "secret not found" error, too. 

What happened to me was:
1. use dockerConfig 1 --> job failed (secret already exists)
2. use dockerConfig 2 instead --> job failed (secret not found)
3. use dockerConfig 1 --> job failed (secret already exists)

It'd be much easier to just retry the job.

Reply:
Rewrote the comment.